packages/powerapps-lib/src/schema/processors/__tests__/schema-processes.spec.js

```diff
@@ -50,7 +50,7 @@ describe('the schema processes', () => {
       expect(applicationPayload).toEqual({
         relationshipsPayload: null,
         columnPayload: {
-          fullname: 'Mr Brian Yak',
+          lastname: 'Mr Brian Yak',
```

Comment:
Can we please update the value to better reflect the key?

Reply:
See above